CMakeLists.txt

```diff
@@ -1,22 +1,29 @@
-cmake_minimum_required (VERSION 2.8.12)
+cmake_minimum_required (VERSION 3.25)
```

Comment:
Is there a reason why we need such a recent CMake version?

Reply:
yes, it's to enable the LINUX, BSD variables.
Note that this is the CMake version available in the current Debian Stable.

Reply:
Fair enough!